helm/ngrok-operator/templates/bindings-forwarder/deployment.yaml

```diff
@@ -95,24 +95,10 @@ spec:
               fieldPath: metadata.namespace
         - name: HELM_RELEASE_NAME
           value: {{ .Release.Name | quote }}
-        - name: SSL_CERT_DIR
-          value: /etc/ssl/certs/ngrok
         {{- range $key, $value := .Values.extraEnv }}
         - name: {{ $key }}
           value: {{- toYaml $value | nindent 12 }}
         {{- end }}
-        {{- if .Values.extraVolumeMounts  }}
```

Comment:
I think we still want to support adding extra volume mounts, we just don't want to stick the `ngrok-ca` one in here now